Lib/AssetScanner.php

```diff
@@ -233,4 +233,13 @@ public function isRemote($target) {
 		return false;
 	}
 
+/**
+ * Gets webroot path
+ *
+ * @return string
+ */
+	protected function _getWebroot() {
+		return WWW_ROOT;
+	}
```

Comment:
That isn't the change that makes fixes it, it's there for testing purposes. A bit further down that line, `$path` is prepended.